solr/core/src/test/org/apache/solr/cloud/SSLMigrationTest.java

```diff
@@ -63,7 +61,7 @@ public void test() throws Exception {
     testMigrateSSL(new SSLTestConfig(false, false));
   }
   
-  public void testMigrateSSL(SSLTestConfig sslConfig) throws Exception {
+  private void testMigrateSSL(SSLTestConfig sslConfig) throws Exception {
```

Comment:
?

Reply:
it's an internal test method and doesn't need to be public but I'll rename it to make it clear